zkevm-circuits/src/pi_circuit.rs

```diff
@@ -839,131 +842,132 @@ impl<F: Field, const MAX_TXS: usize, const MAX_CALLDATA: usize> Circuit<F>
     }
 }
 
-#[cfg(test)]
-mod pi_circuit_test {
-    use super::*;
-
-    use crate::test_util::rand_tx;
-    use halo2_proofs::{
-        dev::{MockProver, VerifyFailure},
-        halo2curves::bn256::Fr,
-    };
-    use pretty_assertions::assert_eq;
-    use rand::SeedableRng;
-    use rand_chacha::ChaCha20Rng;
-
-    /// Compute the raw_public_inputs column from the verifier's perspective.
-    fn raw_public_inputs_col<F: Field, const MAX_TXS: usize, const MAX_CALLDATA: usize>(
-        public_data: &PublicData,
-        randomness: F, // For RLC encoding
-    ) -> Vec<F> {
-        let block = public_data.get_block_table_values();
-        let extra = public_data.get_extra_values();
-        let txs = public_data.get_tx_table_values();
+#[cfg(any(feature = "test", test))]
```

Comment:
Would drop the `test` config if we use it externally.

Reply:
dropped!